packages/dialtone-css/gulpfile.cjs

```diff
@@ -108,6 +109,10 @@ const paths = {
     inputLib: './lib/build/less/dialtone.less',
     outputLib: './lib/dist/',
   },
+  tokens: {
+    input: 'node_modules/@dialpad/dialtone-tokens/dist/css/*.css',
```

Comment:
Wonder if it's safer to reference the relative path to the actual files instead of the installed package.

Reply:
could yeah. do you think that's better? In theory either way should work just as well shouldn't it?

Reply:
Yeah, either way should work. Guess pnpm installed local package is just a soft-link  so it's the same and it's safer this way.